bftengine/src/bftengine/ReplicaLoader.cpp

```diff
@@ -316,7 +316,7 @@ ReplicaLoader::ErrorCode loadReplicaData(shared_ptr<PersistentStorage> p, Loaded
       Verify((e.isCheckpointMsgSet()), InconsistentErr);
       Verify((e.getCheckpointMsg()->seqNumber() == seqNum), InconsistentErr);
       Verify((e.getCheckpointMsg()->senderId() == ld.repConfig.replicaId), InconsistentErr);
-      VerifyOR(seqNum > ld.lastStableSeqNum, e.getCheckpointMsg()->isStableState(), InconsistentErr);
+      VerifyOR(seqNum >= ld.lastStableSeqNum, e.getCheckpointMsg()->isStableState(), InconsistentErr);
```

Comment:
Very cool, thanks Yulia :) I'm quite confident this will put an end to the intermittent failures in persistence testing.

Reply:
Well, apparently, it doesn't:
https://travis-ci.com/vmware/concord-bft/jobs/267559409

We'll have to dig deeper into this, once we enable log4cpp in Travis CI.

Still, your change here is relevant and I suggest to keep it.

Reply:
Hm, the link you gave points to a passed test.

Reply:
This is because I reran the test.